mappings/net/minecraft/world/gen/surfacebuilder/SurfaceRules.mapping

```diff
@@ -78,11 +89,13 @@ CLASS net/minecraft/unmapped/C_flergppe net/minecraft/world/gen/surfacebuilder/S
 		FIELD f_knsaxicl preliminarySurfaceCache [I
 		FIELD f_koxwamhv minSurfaceLevel I
 		FIELD f_lvzwbbpb temperatureCondition Lnet/minecraft/unmapped/C_flergppe$C_vjorfddw;
+		FIELD f_ngmzwvpj HOW_FAR_BELOW_PRELIMINARY_SURFACE_LEVEL_TO_BUILD_SURFACE I
```

Comment:
HOW_FAR_BELOW_PRELIMINARY_SURFACE_LEVEL_TO_BUILD is what I believe it should be instead. Up to you though.